game_eggs/minecraft/java/ftb/egg-ftb-modpacksch-server.json

```diff
@@ -13,23 +13,23 @@
         "java_version",
         "pid_limit"
     ],
-    "images": [
-        "ghcr.io\/pterodactyl\/yolks:java_8",
-        "ghcr.io\/pterodactyl\/yolks:java_11",
-        "ghcr.io\/pterodactyl\/yolks:java_16",
-        "ghcr.io\/pterodactyl\/yolks:java_17"
-    ],
+    "docker_images": {
+        "Java 17": "ghcr.io\/pterodactyl\/yolks:java_17",
+        "Java 16": "ghcr.io\/pterodactyl\/yolks:java_16",
+        "Java 11": "ghcr.io\/pterodactyl\/yolks:java_11",
+        "Java 8": "ghcr.io\/pterodactyl\/yolks:java_8"
+    },
     "file_denylist": [],
-    "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar forge-server.jar",
+    "startup": "java -javaagent:log4jfix\/Log4jPatcher-1.0.0.jar -Xms128M -XX:MaxRAMPercentage=95.0 -Dterminal.jline=false -Dterminal.ansi=true $( [[  ! -f unix_args.txt ]] && printf %s \"-jar start-server.jar\" || printf %s \"@unix_args.txt\" )",
```

Comment:
does this use unix_args? and is the log4jfix/Log4jPatcher-1.0.0.jar file always present with that version?( maybe use a wildvard for the version)

Reply:
Yes it launches with unix_args.txt, unless that doesn't exist, then it launches with start-server.jar (for either fabric or pre-1.17 forge)

And the installer always downloads version 1.0.0 of Log4jPatcher, from here: https://github.com/CreeperHost/modpacksch-serverdownloader/blob/fba9792cf71889ce973a6a0846a194e3113e1678/main.go#L405
However trying to use a wildcard for the version prevents the server from starting:
```
container@pterodactyl~ java -javaagent:log4jfix/Log4jPatcher-*.jar -Xms128M -XX:MaxRAMPercentage=95.0 -Dterminal.jline=false -Dterminal.ansi=true @unix_args.txt
Error opening zip file or JAR manifest missing : log4jfix/Log4jPatcher-*.jar
Error occurred during initialization of VM
agent library failed to init: instrument
container@pterodactyl~ Server marked as offline...
[Pterodactyl Daemon]: ---------- Detected server process in a crashed state! ----------
[Pterodactyl Daemon]: Exit code: 1
[Pterodactyl Daemon]: Out of memory: false
[Pterodactyl Daemon]: Aborting automatic restart, last crash occurred less than 60 seconds ago.
```

Reply:
please try `-javaagent:log4jfix/Log4jPatcher-*.*.*.jar`

Reply:
That doesn't work either. Should I just edit the script to add `mv log4jfix/Log4jPatcher-*.jar log4jfix/Log4jPatcher.jar` and change the startup to `-javaagent:log4jfix/Log4jPatcher.jar`, or should it just be left as 1.0.0?

Reply:
Judt leave it like this. It is oke. Then we will judt update the egg when they change it.